Changes between KDE 3.5.10 tag and KDE 3.5 branch r882976.
Only looking at subdirectory kdepim/libkmime here.
Logs since r849627 of the branch but this might be wrong.

r882976 | dfaure
Fix crash after "kontact: WARNING: Base64Decoder: unexpected padding character in input stream". ...
  M kmime_util.cpp


Index: kmime_util.cpp
===================================================================
--- kdepim/libkmime/kmime_util.cpp	 (.../tags/KDE/3.5.10)	(revision 850549)
+++ kdepim/libkmime/kmime_util.cpp	 (.../branches/KDE/3.5)	(revision 882976)
@@ -195,8 +195,10 @@
           {
             str = KCodecs::base64Decode(str);
           }
-          for (i=0; str[i]; i++) {
-            *dest++ = str[i];
+          if (!str.isNull()) {
+            for (i=0; str[i]; i++) {
+              *dest++ = str[i];
+            }
           }
         }
 
